codespell_lib/data/dictionary.txt

```diff
@@ -10287,6 +10287,9 @@ developpers->developers
 developpment->development
 develp->develop
 develped->developed
+develper->developer
+develpers->developers
+develping->developing
 develpment->development
 develps->develops
```

Comment:
It would be great if you can add this. https://github.com/search?q=develpments&type=code
```text
develpments->developments
```

Reply:
[c33a5ba](https://github.com/codespell-project/codespell/pull/2383/commits/c33a5ba54e2c1aa85eb9b525919daaf41b3ce94b)